### ISSUE TITLE
Recipe issue

### ISSUE THREAD

**Issue by Vyrgax** [NyaaCat/RPGitems-reloaded#311]
When Im trying to delete a recipe with /rpgitem (name) removerecipe I get this in console, and the recipe wont be deleted:
https://hastebin.com/oqaboribev.md

Also, can someone explain how the /rpgitem (name) recipe works? I enter 1 as 'chance integer', but I don't know what it does.

**Comment by Librazy** [comment]
Recipe related feature is buggy in 1.12.* due to some API breakage and I'm finding way to fix them.

**Comment by Kristalyze** [comment]
I have some issues with recipe too, I created a recipe for a item using a rpgitems tool whitch is damaged and unbreakable(to get custom textures) with a custom name. But when I try to craft the item, I get the damaged unbreakable item with no custom name back, instead of the one whitch I created the craft for.

Also, not every recipe works. For example, I created 3 different rpgitem swords for the 3 different way of crafting a sword. After setting up every recipe only the one on the far right doesn't work, and it's the same for shovel or axe.
Also, when creating recipe, I only used the far left 3*3 square of the interface.
I hope you will do your best to take care of thoses one too ;)
Thank you!

P.S. Sorry for my poor English, it's not my most used language x/
